modules/ROOT/pages/access-nosql-databases.adoc

```diff
@@ -13,18 +13,17 @@
 :page-type: general
 = Access NoSQL databases with CDI
 
-Not only SQL (NoSQL) databases provide flexible schemas for potentially large amounts of structured, semi-structured, and unstructured data.
-You can access a NoSQL database with a Contexts and Dependency Injection (CDI) producer in Open Liberty, without enabling features like MongoDB or CouchDB.
+In Open Liberty, you can create a CDI producer and configure access to any version of a NoSQL Database in your application.
+Only certain versions of NoSQL databases could be accessed previously, which needed enabling specific features in the server.xml file.
```

Comment:
suggested edit
Only certain versions of NoSQL databases could be accessed previously, which needed enabling specific features in the server.xml file.
--->
In Open Liberty 19.0.0.6 and earlier, only certain versions of NoSQL databases were available, and these required you to enable specific features in your `server.xml` file.